ru/turing/courses/lesson6/filippov/ddl.sql

```diff
@@ -0,0 +1,39 @@
+CREATE TABLE users
+(
+    Id Serial PRIMARY KEY,
+    Avatar bytea,
+    Description varchar(250),
+    Username varchar(50),
+    Fullname varchar(150),
+    Email varchar(100),
+    Password varchar(100),
+    Age INTEGER,
+    CHECK(Age>0 AND Age<=130 AND (Email!='') AND(Username!='')AND(Fullname!='')),
+    UNIQUE(Username,Email)
+);
+CREATE TABLE article
+(
+    Id Serial PRIMARY KEY,
+    IdUser Serial REFERENCES users(Id),
+    Title varchar(200) NOT NULL,
+    Text text NOT NULL,
```

Comment:
Лучше не использовать зарезервированные слова постгреса, замени на
```suggestion
    content text NOT NULL,
```